{
  "id": "EUVD-2020-20053",
  "description": "libdwarf before 20201017 has a one-byte out-of-bounds read because of an invalid pointer dereference via an invalid line table in a crafted object.",
  "datePublished": "Apr 15, 2023, 12:00:00 AM",
  "dateUpdated": "Aug 4, 2024, 4:18:44 PM",
  "baseScore": 6.5,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H",
  "references": [
    "https://www.suse.com/security/cve/CVE-2020-27545.html",
    "https://sourceforge.net/projects/libdwarf/",
    "https://github.com/davea42/libdwarf-code/commit/95f634808c01f1c61bbec56ed2395af997f397ea",
    "https://www.prevanders.net/dwarfbug.html#DW202010-001",
    "http://web.archive.org/web/20190601140703/https://sourceforge.net/projects/libdwarf/",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2025694"
  ],
  "aliases": [
    "CVE-2020-27545",
    "GSD-2020-27545"
  ],
  "assigner": "mitre",
  "epss": 0.06,
  "enisaIdProduct": [
    {
      "id": "de8bfaaf-1480-39ca-a8d4-fa42f7b557d0",
      "product": {
        "name": "n/a"
      },
      "product_version": "n/a"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "01ff6ba2-82ee-3450-935b-8ba819a214ae",
      "vendor": {
        "name": "n/a"
      }
    }
  ],
  "cve": "CVE-2020-27545"
}